app/plugins/Awards/tests/TestCase/Model/Table/RecommendationsTableTest.php

```
@@ -0,0 +1,110 @@
+            $this->buildRecommendationData($activeAward->id, 'keep-inactive-existing-' . uniqid('', true)),
+        ));
+
+        $activeAward->is_active = false;
+        $this->Recommendations->Awards->saveOrFail($activeAward);
+
+        $recommendation->reason = 'keep-inactive-existing-updated-' . uniqid('', true);
+
+        $savedRecommendation = $this->Recommendations->save($recommendation);
+
+        $this->assertNotFalse($savedRecommendation);
+        $this->assertSame($activeAward->id, $savedRecommendation->award_id);
```

PHPStan (2.1.51)

[error] Cannot access property $award_id on Awards\Model\Entity\Recommendation|false.


(property.nonObject)